MAUI/Picker/date-time-picker.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: DateTimePicker with .NET MAUI Picker control | Syncfusion
+description: Learn here all about date time picker with Syncfusion .NET MAUI Picker (SfPicker) control.
+platform: maui
+control: SfDateTimePicker
+documentation: ug
+---
+
+# DateTime Picker in .NET MAUI Picker (SfDateTimePicker)
+
+This section explains about the time picker.
+
+## DateTimePicker customization
+
+The `SfDateTimePicker` supports to customize [HourInterval], [MinuteInterval], [SecondInterval], [DateFormat], [TimeFormat], [MinimumDate], [MaximumDate], [DayInterval], [MonthInterval], [YearInterval], [SelectedDate] properties by the following code.
```

Comment:
The `SfDateTimePicker` supports customization of the following properties: [HourInterval], [MinuteInterval], [SecondInterval], [DateFormat], [TimeFormat], [MinimumDate], [MaximumDate], [DayInterval], [MonthInterval], [YearInterval], and [SelectedDate]. This can be achieved using the following code.

Reply:
Content modified